src/components/AppHeader.tsx

```diff
@@ -0,0 +1,61 @@
+import { AppBar, Avatar, Box, Stack, Typography } from '@mui/material'
+import { getBearerSelector, getUserDataSelector } from '../store/selectors'
+import { useDispatch, useSelector } from 'react-redux'
+import { DropMenu } from './DropMenu'
+import { RootState } from '../store/store'
+import { AuthorizationButton } from './AuthorizationButton'
+import { useNavigate } from 'react-router-dom'
+import { setBearerAction } from '../store/slices/bearerSlice'
+import { setUserAction } from '../store/slices/userSlice'
+import { UnknownResponseData } from '../types'
+
+export const AppHeader = () => {
+  const data = useSelector<RootState, UnknownResponseData>(getUserDataSelector)
```

Comment:
Please always try to use generics when you don't know type

Reply:
fixed